parts/k8s/kubernetesconfigscustomcloud.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+AZURE_STACK_ENV="azurestackcloud"
+function ensureCertificates() {
+    if [[ "${TARGET_ENVIRONMENT,,}" == "${AZURE_STACK_ENV}" ]]; then 
```

Comment:
We don't need this `if` check, as the only caller of this function is `configureK8sCustomCloud`, which is already guarded by the same `if` check.